custom_model_runner/datarobot_drum/drum/gpu_predictors/utils.py

```diff
@@ -107,7 +107,7 @@ def _configure_ngc_client(self):
 class S3Client:
     def __init__(self, s3_url, credential):
         import boto3
-        from mlpiper.extra.aws_helper import AwsHelper
+        from .aws_helper import AwsHelper
```

Comment:
Why do we need this import to happen inside of the class? Would there be any conflicts otherwise?

Reply:
My only guess is that this class requires `boto3` installed and I'm not sure if that is a required dependency of `DRUM`? If it is required then we can move this import to the top level, if it is not required then I'd prefer leaving this import to be _just in time_